src/node_contextify.cc

```diff
@@ -844,59 +842,35 @@ void ContextifyScript::CreateCachedData(
   }
 }
 
-void ContextifyScript::RunInThisContext(
-    const FunctionCallbackInfo<Value>& args) {
-  Environment* env = Environment::GetCurrent(args);
-
-  ContextifyScript* wrapped_script;
-  ASSIGN_OR_RETURN_UNWRAP(&wrapped_script, args.Holder());
-
-  TRACE_EVENT0(TRACING_CATEGORY_NODE2(vm, script), "RunInThisContext");
```

Comment:
This change removes the `RunInThisContext` trace event but it's probably fine because trace events is still considered experimental. The name also needs to be removed from the list in this test - https://github.com/nodejs/node/blob/7ad5b420aebc1bcd07be187481cf48a4180a60c0/test/parallel/test-trace-events-vm.js#L11.

Reply:
Thanks for the catching. I removed the trace name from the test and also removed the function declaration I missed.

Reply:
> This change removes the RunInThisContext trace event but it's probably fine because trace events is still considered experimental.

I agree with you.

Plus, some trace event names look to be named by following the internal c++ function name. So I'm curious whether such names need to be guaranteed not to be modified or removed even in the future.

`node:trace_events` module provides only a way to enable/disable logging of categorized trace data. It doesn't seem to strictly force what trace name should be logged.

Reply:
Yea, I agree that the trace event names shouldn't follow semver rules because as you said the c++ functions aren't part of the public API. This would probably get discussed before the tracing subsystem becomes stable.